text/0000-add-geo-processing-abilities-to-skylark.md

```diff
@@ -0,0 +1,411 @@
+- Feature Name: Add geo processing abilities to skylark 
+- Start Date: 2018-10-02
+- RFC PR: <!-- (leave this empty) -->
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+
+# Motivation
+[motivation]: #motivation
+
+A lot of open public data, environmental data, government data etc has some kind of geospatial properties. Being able to 
+perform geospatial operations on these datasets would be a huge win for qri and could help increase adoption of qri as a means
+to distribute these types of dataset
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+When working with spatial datasets, those associated with a point, line, polygon etc we often need to be able to perform manipulations
+of the data based on those geometries. The Geo module of sklark provides the basic data types and methods to be able to do just this.
```

Comment:
Typo: "sklark" -> "skylark". But also, we're soon switching to the new name "starlark".